sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusReactorReceiver.java

```diff
@@ -236,8 +247,11 @@ protected Message decodeDelivery(Delivery delivery) {
     private Mono<Void> updateDispositionInternal(String lockToken, DeliveryState deliveryState) {
         final Delivery unsettled = unsettledDeliveries.get(lockToken);
         if (unsettled == null) {
-            logger.warning("entityPath[{}], linkName[{}], deliveryTag[{}]. Delivery not found to update disposition.",
-                getEntityPath(), getLinkName(), lockToken);
+
+            logger.atWarning()
+                // TODO: it used to be deliveryTag, is it ok to change?
+                .addKeyValue(LOCK_TOKEN_KEY, lockToken)
```

Comment:
@conniey `lockToken` used to be logged under `deliveryTag` key - is it ok to change? thanks!

Reply:
Ahh, delivery tag is the actual AMQP concept.. but end users know it as "lock token". I am fine with this change. It'll help us correlate.